integration-cli/docker_api_logs_test.go

```diff
@@ -82,3 +86,96 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
```

Comment:
That's quite hard to do because, as it stands, the stream is not terminated when the `until` limit is reached. No logs are further outputted, but the stream remains open until the use sends an interrupt signal. 

I've updated the test to grab everything up until the expected cut-off point and then verify that their timestamps.